hass/configurations.js

```diff
@@ -154,9 +154,9 @@ module.exports = {
       brightness_state_topic: true,
       brightness_command_topic: true,
       on_command_type: 'first',
-      state_value_template: '{{ "off" if value_json.value == 0 else "on" }}',
+      state_value_template: '{{ "OFF" if value_json.value == 0 else "ON" }}',
```

Comment:
This is incorrect. 

https://github.com/home-assistant/core/blame/dev/homeassistant/components/mqtt/light/schema_template.py#L223-L226

For 3 years this has been `on` and `off`.

Reply:
I submitted a fix for it with #289, just waiting for the merge :)

Reply:
Looking at those lines in the actual code, @robertsLando, this is fuzzier than expected, it appears inconsistent in the code, and the docs may not be clear/accurate. 

This worked for me in testing, but could be because my RGB Lights specifically are expecting payload_on/off to be all caps. I do not have any normal on/off lights.

I am out of town, and down for whatever makes the most sense. IMO, If this is breaking a lot of peoples stuff, we should revert the change and then open an issue with home-assistant core to investigate further.

Reply:
I will submit a patch release in next ours